### ISSUE TITLE
ArrayDeclarationSniff – doesn't handle short array syntax?

### ISSUE THREAD

**Issue by glen-84** [squizlabs/PHP_CodeSniffer#731]
This code:

```php
<?php

$y = [
    1,
    2
];

$y = [
    1 => 'x',
    2 => 'y'
];
```
Shows the following errors:

      4 | ERROR | [x] Array value not aligned correctly; expected 6
        |       |     spaces but found 4
      5 | ERROR | [x] Comma required after last value in array
        |       |     declaration
      6 | ERROR | [x] Closing parenthesis not aligned correctly; expected
        |       |     5 space(s) but found 0
      9 | ERROR | [x] Array key not aligned correctly; expected 6 spaces
        |       |     but found 4
     10 | ERROR | [x] Array key not aligned correctly; expected 6 spaces
        |       |     but found 4
     11 | ERROR | [x] Closing parenthesis not aligned correctly; expected
        |       |     5 space(s) but found 0

**Comment by aik099** [comment]
1. Which version PHP_CodeSniffer are you using?
2. If `ArrayDeclarationSniff` sniff can be found in multiple standards, then which one of them are you using?

**Comment by glen-84** [comment]
1. 2.3.4
2. Squiz.Arrays.ArrayDeclaration

**Comment by gsherwood** [comment]
The sniff does support short arrays and has for quite a while. The errors you are getting are because the sniff is asking you to format the arrays in a specific way (the same way it does for standard arrays), so it wants this:
```php
$y = [
      1,
      2,
     ];

$y = [
      1 => 'x',
      2 => 'y',
     ];
```
If you don't format your arrays like that, then it isn't the right sniff for you.

**Comment by glen-84** [comment]
Ah, my apologies. We don't format our arrays like that.
